src/DataFormatter.php

```diff
@@ -310,7 +311,7 @@ protected function getFieldsForObj($obj)
         }
 
         // add default required fields
-        $dbFields = array_merge($dbFields, array('ID'=>'Int'));
+        $dbFields = array_merge($dbFields, array('ID' => 'Int'));
```

Comment:
Super low impact and not generally a thing to worry about eliminating from existence everywhere, but when we're updating code we should generally try to update to the new array syntax: `[` ... `]`